pkgs/sketch_pad/lib/widgets.dart

```diff
@@ -124,7 +124,7 @@ class RunButton extends StatelessWidget {
               size: 20.0,
             ),
             SizedBox(
-              width: 8.0,
+              width: 16.0,
```

Comment:
👍

Reply:
https://github.com/dart-lang/dart-pad/pull/2870/files#diff-ed60136a3d9ee7c71a9ed356332807808330f0cb99e3959050cfe26b087d6f8eR146 . I believe this PR has been merged now and I have resolved the merge conflicts wrt main.

Reply:
that surfaceVariant check is again failing

Reply:
so do I need to wait , before this depreciation issue is fixed with some other PR?